backend/repository/thrift_update.go

```diff
@@ -45,6 +45,39 @@ func (r *Repository) ThriftConfig(idlRoot string) (map[string]*ThriftMeta, error
 	return config, nil
 }
 
+// WriteManagedThriftFiles will write new thrift files to disk
+func (r *Repository) WriteManagedThriftFiles(files []ManagedThriftFile) error {
```

Comment:
would be nice to throw some trackable errors, `NewRequestError( mfile.filename)` maybe?

Reply:
sure, will change.